src/messages/formatters.ts

```diff
@@ -146,6 +146,13 @@ export function hyperlink(content: string, url: string | URL, title?: string) {
 	return title ? `[${content}](${url} "${title}")` : `[${content}](${url})`;
 }
 
+/**
+ * Wraps the content inside spoiler (hidden text).
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function spoiler<C extends string>(content: C): `||${C}||`;
```

Comment:
You forgot to implement the function

Reply:
My bad, https://github.com/discordjs/builders/pull/16/commits/9c5326bef8a34d9e1cf128bfaf32ce97cbf32e0a should have fixed it